src/Shared/StorageUtility.ts

```diff
@@ -44,7 +44,11 @@ export class SessionStorageUtility {
   }
 
   public static getEntryNumber(key: StorageKey): number {
-    return StringUtility.toNumber(localStorage.getItem(StorageKey[key]));
+    return StringUtility.toNumber(sessionStorage.getItem(StorageKey[key]));
```

Comment:
Is there anywhere relying on this being `localStorage`?

Reply:
nope, this isn't being called anywhere. safe to change